app/src/main/java/com/example/touchcar/presentation/choose_model/recycler/ChooseModelDiffCallback.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.touchcar.presentation.choose_model.recycler
+
+import androidx.recyclerview.widget.DiffUtil
+import com.example.touchcar.domain.entity.Manufacturer
+import com.example.touchcar.domain.entity.Market
+import com.example.touchcar.domain.entity.Model
+
+class ChooseModelDiffCallback(private val oldList: List<Model>, private val newList: List<Model>) :
+    DiffUtil.Callback() {
```

Comment:
Если переносишь супер-класс на отдельную строку, то переноси и `) :`, чтобы окончание сигнатуры класса было на одной строке:
```suggestion
class ChooseModelDiffCallback(
    private val oldList: List<Model>, 
    private val newList: List<Model>
) : DiffUtil.Callback() {
```